src/test/java/lotto/services/LottoStatisticsServiceTest.java

```diff
@@ -0,0 +1,87 @@
+package lotto.services;
+
+import lotto.models.Lotto;
+import lotto.models.LottoStatistics;
+import lotto.models.enums.Rank;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LottoStatisticsServiceTest {
+
+    private final static LottoStatisticsService lottoStatisticsService = new LottoStatisticsService();
+    private final static List<Lotto> lottos = new ArrayList<>();
+    private final static String WINNING_NUMBER = "1,2,3,4,5,6";
+    private final static List<Lotto> duplicatedLottos = new ArrayList<>();
+    private final static int PAYMENT = 3500;
+
+    @BeforeAll
+    static void setLotto() {
+        lottos.add(Lotto.of(List.of(1,2,3,4,5,6)));
+        lottos.add(Lotto.of(List.of(1,2,3,7,8,9)));
+        lottos.add(Lotto.of(List.of(11,12,13,14,15,16)));
+    }
+
+    @BeforeAll
+    static void setDuplicatedLottos() {
+        duplicatedLottos.add(Lotto.of(List.of(1,2,3,4,5,6)));
+        duplicatedLottos.add(Lotto.of(List.of(1,2,3,4,5,6)));
+        duplicatedLottos.add(Lotto.of(List.of(1,2,3,4,5,6)));
+    }
```

Comment:
`@BeforeAll` 을 나눈 이유가 있을까요? 🤔

```suggestion
    @BeforeAll
    static void setLotto() {
        lottos.add(Lotto.of(List.of(1,2,3,4,5,6)));
        lottos.add(Lotto.of(List.of(1,2,3,7,8,9)));
        lottos.add(Lotto.of(List.of(11,12,13,14,15,16)));
    }

    @BeforeAll
    static void setDuplicatedLottos() {
        duplicatedLottos.add(Lotto.of(List.of(1, 2, 3, 4, 5, 6)));
        duplicatedLottos.add(Lotto.of(List.of(1,2,3,4,5,6)));
        duplicatedLottos.add(Lotto.of(List.of(1,2,3,4,5,6)));
    }
```
 스페이스도 컨벤션이 있습니다 IDEA의 auto formatting 기능을 적극 활용하면 좋을 것 같네요.

Reply:
음.. 다른 객체라서 각각 셋팅해 주는게 좋을 것 같아 나눠보았습니다.
ide는 셋팅해보겠습니다.